src/tiered/tiered_cursor.c

```diff
@@ -1064,11 +1065,12 @@ __curtiered_next_random(WT_CURSOR *cursor)
         break;
     }
 
-    /* We have found a valid doc. Set that we are now positioned */
-    if (0) {
+    if (ret == WT_NOTFOUND) {
```

Comment:
Not sure if this adheres to our coding style/expectations. Would it be better to keep the `if (0)` and add a `WT_ERR(ret)` before it?

Reply:
Should it be any `ret != 0`? I guess we cannot get here with ret being anything other than 0 or `WT_NOTFOUND`, otherwise it would have taken the path to the err label.